assets/javascripts/discourse/components/event-rsvp.js.es6

```diff
@@ -67,14 +61,14 @@ export default Ember.Component.extend({
     return false;
   },
 
-  updateTopic(username, action, type) {
+  updateTopic(user_id, action, type) {
```

Comment:
@fzngagan If we're instantiating a variable on the client, the convention is to use camelCase, rather than underscores (which is used on the server).